yarn.lock

```diff
@@ -2,3 +2,27 @@
 # yarn lockfile v1
 
 
+ansi-regex@^5.0.1:
```

Comment:
Pode remover o yarn.lock também, só usamos package.json e package-lock.json.

Em alguma branch tinham colocado a seguinte entrada no gitignore:
*yarn*

Talvez ela não entrou ainda então podem adicionar isso daí e remover todos esses arquivos extras.

Reply:
só agora vi que saiu errado, o github renderizou \*yarn\* como se fosse itálico, mas meu intuito era que fosse um asterisco mesmo.